cqrs-user-query-service/src/main/java/dev/amir/userquery/framework/input/rabbitmq/listener/UserListener.java

```diff
@@ -2,17 +2,23 @@
 
 import dev.amir.userquery.framework.input.rabbitmq.handler.UserMessageHandler;
 import dev.amir.userquery.framework.input.rabbitmq.message.SaveUserMessage;
+import lombok.extern.slf4j.Slf4j;
 import lombok.RequiredArgsConstructor;
 import org.springframework.amqp.rabbit.annotation.RabbitListener;
 import org.springframework.stereotype.Component;
 
+@Slf4j
 @RequiredArgsConstructor
 @Component
 public class UserListener {
     private final UserMessageHandler userMessageHandler;
 
     @RabbitListener(queues = {"${spring.rabbitmq.queues.users-queue}"})
     public void onSaveUser(SaveUserMessage message) {
+        log.info("Received SaveUserMessage: {}.");
```

Comment:
Add the variable message to be printed at the end of the method

`log.info("Handling incoming message: {}.", message);`